### PR TITLE
feat: add your data example

### PR DESCRIPTION
**Pull request by leethater**
Implemented "Add your own data" example.

### CODE REVIEW

File: app/src/main/java/io/jawg/DataMapActivity.kt [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                val geoJsonUrl = "https://media.jawg.io/add-your-data/marketplaces.geojson"
+                val geoJsonSource = GeoJsonSource("geojson-paris-markets", URI(geoJsonUrl))
+
+                // Add it to the map
+                it.addSource(geoJsonSource)
+
+                // Create a layer with the desired style for our source.
+                val layer = SymbolLayer("symbollayer", "geojson-paris-markets")
+                    .withProperties(
+                        PropertyFactory.iconImage(DataMapActivity.MARKER_ICON),
```
Comment (Joxit) [inline app/src/main/java/io/jawg/DataMapActivity.kt]: Replace `DataMapActivity.MARKER_ICON` by `MARKER_ICON`

File: app/src/main/java/io/jawg/DataMapActivity.kt [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            map.setStyle(makeStyleUrl()) {
+                // Map fully loaded in this scope.
+                // Update attributions position
+                map.uiSettings.setAttributionMargins(15, 0, 0, 15)
+
+                // Add an image to use as a custom marker
+                val selectedMarkerIconDrawable =
+                    ResourcesCompat.getDrawable(this.resources, R.drawable.ic_jawg_marker, null)
+                it.addImage(
+                    DataMapActivity.MARKER_ICON,
```
Comment (Joxit) [inline app/src/main/java/io/jawg/DataMapActivity.kt]: Replace `DataMapActivity.MARKER_ICON` by `MARKER_ICON`

File: app/src/main/java/io/jawg/DataMapActivity.kt [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+                // Create a GeoJson Source from our remote geojson.
+                val geoJsonUrl = "https://media.jawg.io/add-your-data/marketplaces.geojson"
+                val geoJsonSource = GeoJsonSource("geojson-paris-markets", URI(geoJsonUrl))
+
+                // Add it to the map
+                it.addSource(geoJsonSource)
+
+                // Create a layer with the desired style for our source.
+                val layer = SymbolLayer("symbollayer", "geojson-paris-markets")
```
Comment (Joxit) [inline app/src/main/java/io/jawg/DataMapActivity.kt]: Rename the layer name from `symbollayer` to `paris-market-pois`

**Review by Joxit — APPROVED** [review]
Minor changes to do before merge

File: app/src/main/java/io/jawg/DataMapActivity.kt (hunk 1 above)
Comment (leethater) [reply app/src/main/java/io/jawg/DataMapActivity.kt]: Done

File: app/src/main/java/io/jawg/DataMapActivity.kt (hunk 2 above)
Comment (leethater) [reply app/src/main/java/io/jawg/DataMapActivity.kt]: Done

File: app/src/main/java/io/jawg/DataMapActivity.kt (hunk 3 above)
Comment (leethater) [reply app/src/main/java/io/jawg/DataMapActivity.kt]: Changed

**Pull request merged**